### PR TITLE
Read CMIS data path state duration

### PR DESCRIPTION
**Pull request by abohanyang**
Read CMIS data path state duration

#### Description
Read page 0x1, reg 144 to get the data path max init/deinit duration. 

#### Motivation and Context
The data path max init duration will replace the CMIS_DEF_EXPIRED in xcvrd.

#### How Has This Been Tested?
build passes

#### Additional Information (Optional)
n/a

### CODE REVIEW

File: sonic_platform_base/sonic_xcvr/mem_maps/public/cmis.py [hunk 1]
```diff
@@ -110,6 +110,12 @@ def __init__(self, codes):
 
         # Should contain ONLY upper page fields
         self.ADVERTISING = RegGroupField(consts.ADVERTISING_FIELD,
+            CodeRegField(consts.DP_PATH_INIT_DURATION, self.getaddr(0x1, 144), self.codes.DP_PATH_TIMINGS,
+                 *(RegBitField("Bit%d" % (bit), bit) for bit in range (0, 4))
+            ),
+            CodeRegField(consts.DP_PATH_DEINIT_DURATION, self.getaddr(0x1, 144), self.codes.DP_PATH_TIMINGS,
+                 *(RegBitField("Bit%d" % (bit), bit) for bit in range (4, 8))
+            ),
```
Comment (prgeor) [inline sonic_platform_base/sonic_xcvr/mem_maps/public/cmis.py:118]: The first offset in the RegGroupField should be the smallest among all the fields inside the RegGroupField. Ref:- [https://github.com/sonic-net/sonic-platform-common/blob/master/sonic_platform_base/sonic_xcvr/fields/xcvr_field.py#L227](https://github.com/sonic-net/sonic-platform-common/blob/master/sonic_platform_base/sonic_xcvr/fields/xcvr_field.py#L227)

**Comment by prgeor** [comment]
@abohanyang can you add code coverage?

File: sonic_platform_base/sonic_xcvr/mem_maps/public/cmis.py (hunk 1 above)
Comment (abohanyang) [reply sonic_platform_base/sonic_xcvr/mem_maps/public/cmis.py]: I address this in the latest commit.

**Review by prgeor — APPROVED** [review]

**Comment by prgeor** [comment]
@shyam77git FYI

**Review by prgeor — APPROVED** [review]

**Pull request merged**